WORKSPACE

```diff
@@ -300,14 +300,11 @@ cc_library(
 )
 """
 
-http_archive(
+git_repository(
     name = "zlib",
-    build_file_content = _zlib_build,
-    sha256 = "38ef96b8dfe510d42707d9c781877914792541133e1870841463bfa73f883e32",
-    strip_prefix = "zlib-1.3.1",
-    # Using the .tar.xz artifact from the release page – for many other dependencies we use a
-    # snapshot based on the tag of a release instead.
-    urls = ["https://github.com/madler/zlib/releases/download/v1.3.1/zlib-1.3.1.tar.xz"],
+    build_file_content=_zlib_build,
+    commit = "71660e196bcfc74b2c657a46b512037cb49079a2",
```

Comment:
Is this the commit that we use on internal repo? Keeping this value in sync with the V8 version might be a challenge.

Reply:
In edgeworker we link against the object files that v8 already built for zlib rather than the source, but we have several files worth of scaffolding for this. I got the commit hash from the v8 repo and indeed it's going to get annoying to keep it in sync. But it's the same problem with ICU isn't it?